packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -162,7 +162,11 @@ typedef NestedScrollViewHeaderSliversBuilder = List<Widget> Function(BuildContex
 ///
 // See https://github.com/flutter/flutter/issues/54059
 /// Currently, [NestedScrollView] does not support stretching the outer
-/// scrollable, e.g. when using [SliverAppBar.stretch].
+/// scrollable by default, e.g. when using [SliverAppBar.stretch].
+///
+/// If need to support stretching the outer scrollable, you need to set the
+/// bouncing scroll physics to NestedScrollView.physics, and set
+/// ClampingScrollPhysics to the body's scroll view.
```

Comment:
There should also be some sample code, like the others have for various SliverAppBar effects above.

Reply:
Thank you for the reminder. I have added sample code in nested_scroll_view.3.dart and moved the test to slivers.appbar_stretch_test.dart.